clouddriver-kubernetes/src/test/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/security/KubernetesV1CredentialsSpec.groovy

```diff
@@ -1,28 +1,30 @@
 /*
- * Copyright 2016 Netflix, Inc.
+ * Copyright 2017 Google, Inc.
```

Comment:
File was added in 2015, but moved (with the wrong intellij settings most likely)